src/components/clusters/ClusterDetailsMultiRegion/components/MachinePools/components/EditMachinePoolModal/fields/__tests__/WindowsLicenseIncludedField.test.tsx

```diff
@@ -0,0 +1,193 @@
+import * as React from 'react';
+import { Formik, FormikValues } from 'formik';
+
+import links from '~/common/installLinks.mjs';
+import { render, screen } from '~/testUtils';
+
+import { WindowsLicenseIncludedField } from '../WindowsLicenseIncludedField';
+
+import {
+  initialValues,
+  initialValuesEmptyMachineType,
+  initialValuesWithWindowsLIEnabledMachineTypeSelected,
+  WindowsLIDisabledMachinePool,
+  WindowsLIEnabledMachinePool,
+} from './WindowsLicenseIncludedField.fixtures';
+
+const {
+  WINDOWS_LICENSE_INCLUDED_AWS_DOCS: AWS_DOCS_LINK,
+  WINDOWS_LICENSE_INCLUDED_REDHAT_DOCS: REDHAT_DOCS_LINK,
+} = links;
+
+// Formik Wrapper:
+const buildTestComponent = (
+  initialValues: FormikValues,
+  children: React.ReactNode,
+  onSubmit: () => void = jest.fn(),
+  formValues = {},
+) => (
+  <Formik
+    initialValues={{
+      ...initialValues,
+      ...formValues,
+    }}
+    onSubmit={onSubmit}
+  >
+    {children}
+  </Formik>
+);
+
+describe('<WindowsLicenseIncludedField />', () => {
+  describe('When creating a new Machine Pool', () => {
+    describe('When selected Machine Type is Windows LI compatible', () => {
+      it('Shows the checkbox text', async () => {
```

Comment:
this function doesn't need to be async; please remove the `async` modifier here, as well as in other similar functions in this module

Reply:
`async` removed